cmd/ocm-support/add/cmd.go

```diff
@@ -0,0 +1,18 @@
+package add
+
+import (
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/add/instances"
+	"github.com/spf13/cobra"
+)
+
+// Cmd ...
+var Cmd = &cobra.Command{
+	Use:   "add [COMMAND]",
+	Short: "Adds the given resource",
+	Long:  "Adds the given resource",
+	Args:  cobra.MinimumNArgs(1),
+}
+
+func init() {
+	Cmd.AddCommand(instances.Cmd)
```

Comment:
I'm wondering if `add` is the best name for this - in reality, we can use this script to `update` the list of cloud resources in AMS.

Reply:
As discussed, changed to ocm support sync-cloud-resources